core_lib/src/interface/timelinecells.cpp

```diff
@@ -425,8 +425,20 @@ void TimeLineCells::paintEvent(QPaintEvent*)
         }
 
         // --- draw the position of the current frame
-        if (mEditor->currentFrame() > mFrameOffset)
+        if (mEditor->playback()->isPlaying() && mEditor->currentFrame() < mFrameOffset)
         {
+            mFrameOffset = mEditor->currentFrame() - 1;
+            emit offsetChanged(mFrameOffset);
+            mTimeLine->updateContent();
+        }
+        else
+        {
+            if (mEditor->playback()->isPlaying() && width() < (mEditor->currentFrame() - mFrameOffset + 1) * mFrameSize)
```

Comment:
The code here could be tied up by putting it into a separate function, calling it trackScrubber() or something similar.

Reply:
Moved tracking to separate function. It seem to work like before.